### ISSUE TITLE
crdc website resources changes

### ISSUE THREAD

**Issue by ndricca** [darribas/gds19#4]
Hi Dani, 

first of all, thank you for your work. 
I started studying by myself PySAL and decided to follow your lessons inside this repo.

I just want to let you know that CRDC changed the website layout and all your links results now not working to the right resource.
I spent some time looking for the same resources on the renewed website but I was not able to find them.
I noticed that the old version is still available with the following [url](https://archive.data.cdrc.ac.u), so for example I was able to download required resources simply changing the link from [the renewed website](https://data.cdrc.ac.uk/dataset/cdrc-2011-census-data-packs-for-local-authority-district-liverpool-e08000012) to [the archived website](https://archive.data.cdrc.ac.uk/dataset/cdrc-2011-census-data-packs-for-local-authority-district-liverpool-e08000012).

I thought it might interest you, in case you did not already know it.

Thanks again,
Andrea

**Comment by skytwosea** [comment]
Andrea - so glad you found those links; I thought I was being daft.
Looks like the only dataset we can't access is the geo-referenced tweets; I can't follow any VITAL links - I suppose that requires a course registration?

**Comment by ndricca** [comment]
Yes, I think that resource is not publicly available. I'm happy I've helped you :-)
